docs/DeveloperGuide.md

```diff
@@ -168,19 +168,23 @@ The sequence diagram of the DeleteGradeCommand is shown below.
 
 ![DeleteGradeCommand.png](diagrams%2FDeleteGradeCommand.png)
 
-### Storage feature
+### Storage Load feature
 
-#### Implementation
+The `StudentStorage`, `GradeStorage` and `ComponentStorage` classes implement the feature to load data from the 
```

Comment:
Nice explanation of how you used 3 different classes to implement the load data feature